jetty-infinispan/infinispan-remote-query/pom.xml

```diff
@@ -9,7 +9,7 @@
   <name>Jetty :: Infinispan Session Manager Remote with Querying</name>
   <properties>
     <bundle-symbolic-name>${project.groupId}.infinispan.remote.query</bundle-symbolic-name>
-    <infinispan.docker.image.version>9.4.8.Final</infinispan.docker.image.version>
+    <infinispan.docker.image.version>11.0.9.Final</infinispan.docker.image.version>
```

Comment:
use the property `${infinispan.version}`?

Reply:
@olamy I can't use the `infinispan.version` property because there is no docker image at that version.

Reply:
arghhh :(

Reply:
The docker image exists as `11.0.9.Final` ...

https://hub.docker.com/layers/infinispan/server/11.0.9.Final/images/sha256-3dfa71e88775f03cd3a6aa9b7f7618d0518ff10f6ad68b4331d5d4c997bea6be